molecule/default/verify.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Verify role
+  hosts: all
+  become: yes
+
+  tasks:
+
```

Comment:
If you create a `command` task to 
```
curl -sS http://127.0.0.1:9100/metrics | grep '^node_load' 
```
That will tell us whether the exporter is working.

(on centos you may need the specify the full path to curl)